Readme.md

```diff
@@ -170,6 +172,35 @@ ctx.fillText('Everyone hates this font :(', 250, 10)
 
 The second argument is an object with properties that resemble the CSS properties that are specified in `@font-face` rules. You must specify at least `family`. `weight`, and `style` are optional and default to `'normal'`.
 
+### deregisterAllFonts()
+
+> ```ts
+> deregisterAllFonts() => void
+> ```
+
+Use deregisterAllFonts If you registered font and need to remove it, for example if you use canvas in tests.
```

Comment:
```suggestion
Use deregisterAllFonts if you registered font and need to remove it, for example if you use canvas in tests.
```

Reply:
hmm perhaps this description makes it sound like you can deregister a specific font? I believe the api will deregister all previously added fonts

Reply:
@michaeljherrmann do you think it's better?

'Use `deregisterAllFonts` to unregister all fonts that have been previously registered. This method is useful when you want to remove all registered fonts, such as when using the canvas in tests'

Reply:
updated